src/WebJobs.Script/Utils/StringUtils.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Script.Utils
+{
+    internal static class StringUtils
+    {
+        /// <summary>
+        /// Determines whether a delimited string contains a specific token, using the specified separator and string comparison.
+        /// This method is a zero-allocation, faster alternative to splitting the string and using Contains, as it avoids unnecessary allocations.
+        /// </summary>
+        /// <param name="delimitedString">The string containing delimited tokens to search(Ex: "FeatureA,FeatureB").</param>
+        /// <param name="searchToken">The token to search for within the delimited string.</param>
+        /// <param name="separator">The character used to separate tokens in the string. Defaults to ','.</param>
+        /// <param name="comparisonType">The string comparison type to use. Defaults to OrdinalIgnoreCase.</param>
+        /// <returns>True if the token is found; otherwise, false.</returns>
+        internal static bool ContainsToken(string delimitedString, string searchToken, char separator = ',', StringComparison comparisonType = StringComparison.OrdinalIgnoreCase)
```

Comment:
Is it worth taking in `ReadOnlySpan<char>` for the arguments?

```suggestion
        internal static bool ContainsToken(ReadOnlySpan<char> source, ReadOnlySpan<char> token, char separator = ',', StringComparison comparisonType = StringComparison.OrdinalIgnoreCase)
```

Also changed some param names for conciseness.

Maybe even make this an extension method for `string`?

Reply:
Changed param names to concise ones.

Reply:
Switched to extension method. Made another overload which takes ReadOnlySpan<char> as well.